.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.1
```

Comment:
I guess this gem will work on any ruby version higher than 2.x, so fixing version to 2.4.1 is meaningless